mssql-ballerina/Client.bal

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/crypto;
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+# Represents a MsSQL database client.
+public client class Client {
+    *sql:Client;
+    private boolean clientActive = true;
+
+    # Initialize Mssql Client.
+    #
+    # + host - Hostname of the mssql server to be connected
+    # + instanceName - Instance of mssql server to connect to on serverName
```

Comment:
description of `instance` is not clear. check and correct

Reply:
Fixed with [`6ad0fb1`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/6ad0fb1ecfdca2bfc4d43985f5f4adcd7e6ddc37)